translate/convert/l20n2po.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+"""Convert Mozilla .l20n files to Gettext PO localization files.
+
+See: http://docs.translatehouse.org/projects/translate-toolkit/en/latest/commands/l20n2po.html
+for examples and usage instructions.
+"""
+
+from translate.storage import po, l20n
+
+
+class l20n2po:
+    """convert a .l20n file to a .po file for handling the
+    translation."""
+
+    def __init__(self, blankmsgstr=False, duplicatestyle="msgctxt"):
+        pass
+
+    def convertl20nunit(self, store, unit):
+        po_units = []
+
+        po_unit = po.pounit(encoding="UTF-8")
+        po_unit.setid(unit.getid())
+        po_unit.addlocation(unit.getid())
+        po_unit.source = unit.value
+        po_units.append(po_unit)
+
+        return po_units
+
+    def convertstore(self, l20n_store):
+        """converts a .l20n file to a .po file..."""
+        target_store = po.pofile()
+        targetheader = target_store.header()
+        targetheader.addnote("extracted from %s" % l20n_store.filename,
+                             "developer")
+        l20n_store.makeindex()
+        for l20nunit in l20n_store.units:
+            pounits = self.convertl20nunit(l20n_store, l20nunit)
+            for pounit in pounits:
```

Comment:
As said above `pounits` is just a list with a single PO unit inside. I think it is better to simplify this code.